prisma/migrations/20221006161545_create_discussions_table2/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "Discussion" ALTER COLUMN "userPic" DROP NOT NULL;
```

Comment:
Setting userPic to not null, since not every user may have uploaded a profile picture